shuttle/templates/loaded_views/add_user.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'shuttle_base.html' %}
+    <title>Shuttle | {% block page_title %}Add a New User{% endblock %}</title>
+
+{% block body_content %}
+    <h4>Add a New User</h4>
+    <input type="text" id="f-name" placeholder="First Name">
+    <input type="text" id="l-name" placeholder="Last Name">
+    <button type="button" class="btn btn-primary" data-toggle="modal" onclick="searchButton()">Search</button>
+    <br>
+    <div class="dropdown-menu center-page" aria-labelledby="role-dropdown">
+        <a class="dropdown-item" onclick="adminButton()">Administrator</a>
+        <a class="dropdown-item" onclick="driverButton()">Driver</a>
+    </div>
```

Comment:
Center the search fields and button on the page and maybe move the search button to be below the name fields. Maybe something like this 

![Screen Shot 2020-07-14 at 1 03 47 PM](https://user-images.githubusercontent.com/26125143/87460565-ad542b80-c5d2-11ea-9d88-881a4a345f3f.png)